nixos/modules/services/web-servers/keter/config.nix

```diff
@@ -0,0 +1,87 @@
+{ globalketerConfig
+, keterRoot
+, writeShellScript
+, writeTextFile
+, lib
+, bundleCfg
+, callPackage
+, coreutils
+, keter
+, ...
+}:
+
+let
+  incoming = "${keterRoot}/incoming";
+
+
+  globalKeterConfigFile = writeTextFile {
+    name = "keter-config.yml";
+    text = (lib.generators.toYAML { } globalketerConfig);
+  };
+
+  # If things are expected to change often, put it in the bundle!
+  bundle = callPackage ./bundle.nix
+    (bundleCfg // { inherit executable; });
+
+  # This indirection is required to ensure the nix path
+  # gets copied over to the target machine in remote deployments.
+  # Furthermore, it's important that we use exec to
+  # run the binary otherwise we get process leakage.
+  executable = writeShellScript "bundle-wrapper" ''
+    #!/usr/bin/env bash
+    export PATH=$PATH:${lib.makeBinPath [ coreutils ]}
```

Comment:
Still here

Reply:
this was part of the git human/machine error